### ISSUE TITLE
Unable to find a default constructor to use for type FactSet.SDK.FactSetGlobalPrices.Model.Price. Path 'data[0].volume'

### ISSUE THREAD

**Issue by xyzzy-foo-bar-baz** [factset/enterprise-sdk#13]
- **Package Name**: FactSet.SDK.FactSetGlobalPrices
- **Package Version**: 1.2.0
- **Package Language**: NuGet / .NET
- **Operating System**: Windows 11

**Describe the bug**
Calls to `GetGPDPrices` in the OpenAPI auto-generated .NET client fail with error `Unable to find a default constructor to use for type FactSet.SDK.FactSetGlobalPrices.Model.Price. Path 'data[0].volume', line 1, position 19.`

**To Reproduce**

1. Copy the example code at https://github.com/factset/enterprise-sdk/blob/main/code/dotnet/FactSetGlobalPrices/v1/docs/PricesApi.md#getgpdprices
2. Run it with a valid API key which is known-good
3. Error deserializing the response:

```
Exception when calling PricesApi.GetGPDPrices: Unable to find a default constructor to use for type FactSet.SDK.FactSetGlobalPrices.Model.Price. Path 'data[0].volume', line 1, position 19.
Status Code: 500
   at FactSet.SDK.FactSetGlobalPrices.Client.ApiClient.Exec[T](RestRequest req, IReadableConfiguration configuration, RequestOptions requestOptions)
   at FactSet.SDK.FactSetGlobalPrices.Client.ApiClient.Get[T](String path, RequestOptions options, IReadableConfiguration configuration)
   at FactSet.SDK.FactSetGlobalPrices.Api.PricesApi.GetGPDPricesWithHttpInfo(List`1 ids, String startDate, List`1 fields, String endDate, String frequency, String calendar, String currency, String adjust)
   at FactSet.SDK.FactSetGlobalPrices.Api.PricesApi.GetGPDPrices(List`1 ids, String startDate, List`1 fields, String endDate, String frequency, String calendar, String currency, String adjust)
   at FactSetTest.Program.Main(String[] args) in [...].cs:line 30
```

If stepped in a debugger, you can see that the response returns valid JSON data:
```
{"data":[{"volume":42263372,"date":"2019-01-02","priceLow":38.5625,"tradeCount":75417,"requestId":"AAPL-USA","price":39.48,"vwap":38.841488,"fsymId":"C0S0SK-L","currency":"USD","priceOpen":38.700001,"priceHigh":39.709999,"turnover":1641163.56707}, ... ]}
```
etc.

The corp actions endpoint for the same API is working fine for me.

I've also tried generating a new client using what I think is the generator you are using (https://openapi-generator.tech/docs/generators/csharp-netcore/) which behaves the exact same way. So I guess perhaps this is either a bug in that generator, or more likely a bug in your API spec somehow?

**Expected behavior**
GetPrices Result to deserialize and be usable.

**Comment by eschmidtfds** [comment]
Hi @xyzzy-foo-bar-baz ,

thanks for filing this issue. We are looking into this!

**Comment by eschmidtfds** [comment]
Hi @xyzzy-foo-bar-baz we've just released v1.2.1 of the SDK that should fix this error.

The class in question inherited from Dictionary, which confused the JSON parser.
With that removed, it works!

Thanks for helping us making the SDKs better!
Let me know if you have any further questions.

**Comment by eschmidtfds** [comment]
Hi @xyzzy-foo-bar-baz,

since we didnt hear back we'll close this issues as resolved with the release of the fixed SDK.
Please reopen it if you have any other open questions.
